src/middlewared/middlewared/job.py

```diff
@@ -223,6 +223,9 @@ class Job:
     :ivar logs_fd: Unbuffered binary file descriptor for writing logs (if the job was defined with `@job(logs=True)`
     """
 
+    pipes: Pipes
```

Comment:
Just curious, why is this change needed?

Reply:
Sphinx autodoc issues a warning when a documented property (`:ivar` above) is not explicitly declared. We now treat warnings as errors so...